src/Plugin/GraphQL/Derivers/SearchAPIFieldDeriver.php

```diff
@@ -3,18 +3,33 @@
 namespace Drupal\graphql_search_api\Plugin\GraphQL\Derivers;
 
 use Drupal\Component\Plugin\Derivative\DeriverBase;
-use Drupal\search_api\Entity\Index;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Plugin\Discovery\ContainerDeriverInterface;
 use Drupal\graphql_search_api\Utility\SearchAPIHelper;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Provides GraphQL Field plugin definitions for Search API fields.
  */
-class SearchAPIFieldDeriver extends DeriverBase {
+class SearchAPIFieldDeriver extends DeriverBase implements ContainerDeriverInterface {
+
+  /**
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
 
   /**
    * Constructs a new Search API field.
    */
-  public function __construct() {
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
```

Comment:
Same as above we can now include the PHPDoc signature here withe information on the params passed to the constructor.